cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -443,10 +443,13 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 			a.processorCallbacks.disableScaleDownForLoop, scaleDown.nodeDeletionTracker.IsNonEmptyNodeDeleteInProgress())
 
 		if scaleDownInCooldown {
+			metrics.UpdateScalingDownStatus(false)
 			scaleDownStatus.Result = status.ScaleDownInCooldown
 		} else if scaleDown.nodeDeletionTracker.IsNonEmptyNodeDeleteInProgress() {
+			metrics.UpdateScalingDownStatus(false)
```

Comment:
I guess this is fine though no 100% convinced. Maybe we should have non-binary metric (`scale_down_status`)
- 0: scale_down_possbile
- 1: scale_down_in_cooldown
- 2: scale_down_non_empty_node_in_progress

???

Reply:
@MaciekPytel pointed out that having different states coded in value of the metric does not play well (at all) with aggregations on metric. So lets keep you original idea.

Reply:
@losipiuk awesome, I just made the requested changes, let me know what do you think 👍